src/flow.ts

```diff
@@ -25,7 +21,7 @@ import { Note } from './note';
 import { ModifierContext } from './modifiercontext';
 import { MultiMeasureRest } from './multimeasurerest';
 import { TickContext } from './tickcontext';
-import { Articulation } from './articulation';
+import { Articulation, ArticulationStruct } from './articulation';
```

Comment:
lint error: `'ArticulationStruct' is defined but never used`

Reply:
done